src/components/notifications/notifications.service.ts

```diff
@@ -336,6 +357,59 @@ export function getSlackConfiguration(slackConfigId: number, isDeleteComponent?:
     })
 }
 
+export function getWebhookConfiguration(webhookConfigId: number): Promise<ResponseType> {
+    const URL = `${Routes.NOTIFIER}/channel/webhook/${webhookConfigId}`
+    return get(URL)
```

Comment:
we can directly use the value. no need to create variable

Reply:
done